CHANGELOG.md

```diff
@@ -7,122 +7,74 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Fix wrong tx type on edit, by updating tx type in case it changed ([#14721](https://github.com/MetaMask/metamask-extension/pull/14721))
+- Fix chain ID field to use the same color pattern as the currency symbol field warning message ([#14627](https://github.com/MetaMask/metamask-extension/pull/14627))
+- Fix overall CSS for Onboarding V2 flow, making it responsive for large and small screens ([#13902](https://github.com/MetaMask/metamask-extension/pull/13902))
+- Fix performance issue for gridplus-sdk module ([#14582](https://github.com/MetaMask/metamask-extension/pull/14582))
+- Fix currency conversion rate on the Edit tx screen ([#14713](https://github.com/MetaMask/metamask-extension/pull/14713))
+- Fix sign button so it is enabled after scrolling all the way to the bottom ([#14745](https://github.com/MetaMask/metamask-extension/pull/14745))
+- Fix wrong tx type text for `Safe Transaction From` ([#14769](https://github.com/MetaMask/metamask-extension/pull/14769))
+- Fix accidental change on SiteOrigin that removed "chip" on permission screen ([#14768](https://github.com/MetaMask/metamask-extension/pull/14768))
+- Fix "Site Suggested" tooltip in `EditGasFeeBtn` to only display the origin of dapp suggested gas fee if the origin is provided ([#14734](https://github.com/MetaMask/metamask-extension/pull/14734))
+- Fix incorrect line breaks on footer buttons from NetworksTab with nowrap ([#14733](https://github.com/MetaMask/metamask-extension/pull/14733))
+- Fix incorrect currency symbol in the send flow history state logs ([#14726](https://github.com/MetaMask/metamask-extension/pull/14726))
+- Fix batch cancel/reject for signature requests ([#13786](https://github.com/MetaMask/metamask-extension/pull/13786))
+- Fix unrecognized tx type safeTransferFrom, by adding the case in useTransactionDisplayData ([#14678](https://github.com/MetaMask/metamask-extension/pull/14678))
+- Fix "Learn more" dialog broken link from Eth Sign ([#14667](https://github.com/MetaMask/metamask-extension/pull/14667))
+- Fix incorrect casing of includeUsdRate option passed to CurrencyRateController ([#14615](https://github.com/MetaMask/metamask-extension/pull/14615))
+- Fix mixed text for Speed up/Cancel information ([#14551](https://github.com/MetaMask/metamask-extension/pull/14551))
+- Fix SRP paste duplication ([#14625](https://github.com/MetaMask/metamask-extension/pull/14625))
+- Fix yarn audit alert due to cross-fetch vulnerability ([#14570](https://github.com/MetaMask/metamask-extension/pull/14570))
+- Fix browser.tab.get API call by removing `returnToOnboardingInitiator` redundant method ([#14550](https://github.com/MetaMask/metamask-extension/pull/14550))
+- Fix methods like `signTypedData` returning the misleading error `Invalid Address`, by adding `suppressUnauthorized` param to `getAccounts` ([#14126](https://github.com/MetaMask/metamask-extension/pull/14126))
```

Comment:
Suggestion to change "Fix methods like `signTypedData` returning the misleading error `Invalid Address`, by adding `suppressUnauthorized` param to `getAccounts`" to "Fix error handling for signature methods so that proper errors are returned to dapps when there is a permissions error"

Reply:
noted. Ty